samples/wifi/sta/Kconfig

```diff
@@ -8,6 +8,18 @@ source "Kconfig.zephyr"
 
 menu "Nordic Sta sample"
 
+config CLIENT_STATUS_THREAD_STACK_SIZE
```

Comment:
this is a STA sample so, may be use STA as prefix instead of CLIENT or no prefix at all for some?